### PR TITLE
Add SPTDataLoaderExponentialTimerTest

### PR DESCRIPTION
**Pull request by 8W9aG**
* Foot in the door for testing exponential timer
* More tests may follow

### CODE REVIEW

**Comment by 8W9aG** [comment]
@rastersize @jgavris @dflems @JensAyton

File: SPTDataLoaderTests/SPTDataLoaderExponentialTimerTest.m [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    [super setUp];
+    // Put setup code here. This method is called before the invocation of each test method in the class.
+    self.timer = [SPTDataLoaderExponentialTimer exponentialTimerWithInitialTime:1.0 maxTime:10.0];
+}
+
+- (void)tearDown
+{
+    // Put teardown code here. This method is called after the invocation of each test method in the class.
+    [super tearDown];
+}
```
Comment (jgavris) [inline SPTDataLoaderTests/SPTDataLoaderExponentialTimerTest.m]: :gun:

**Comment by jgavris** [comment]
:+1:

**Comment by rastersize** [comment]
:deciduous_tree:

File: SPTDataLoaderTests/SPTDataLoaderExponentialTimerTest.m [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+@end
+
+@implementation SPTDataLoaderExponentialTimerTest
+
+#pragma mark XCTestCase
+
+- (void)setUp
+{
+    [super setUp];
+    // Put setup code here. This method is called before the invocation of each test method in the class.
```
Comment (rastersize) [inline SPTDataLoaderTests/SPTDataLoaderExponentialTimerTest.m]: This comment could be removed as well tbh.

**Pull request merged**